src/app/Fake.Core.Target/Target.fs

```diff
@@ -605,9 +618,34 @@ module Target =
             |> Async.AwaitTask
             |> Async.RunSynchronously
             |> Seq.reduce mergeContext
-
+    module internal Observable =
+        let subscribeOnce callback (observable:IObservable<'T>) =
+            let removeObj : IDisposable option ref = ref None
+            let removeLock = new obj()
+            let setRemover r = 
+                lock removeLock (fun () -> removeObj := Some r)
+                r
+            let remove() = 
+                lock removeLock (fun () -> 
+                    match removeObj.Value with
+                    | Some d -> 
+                        removeObj := None
+                        d.Dispose()
+                    | None -> ())
+            let observer = 
+                { new IObserver<'T> with
+                    member __.OnNext(v) =
+                        remove()
+                        callback v
+                    member __.OnError(_) =
+                        remove()          
+                    member __.OnCompleted()=                           
+                        remove()
+                }
+            observable.Subscribe observer 
```

Comment:
So this is basically `observable.Take(1)`, correct?

Reply:
Yes, should be Take(1)/First. But i did not want to add an extra library

Reply:
I think this has a race when the `OnNext` event is triggered before `setRemover` is called

Reply:
Trying to find a suggestion ...

Reply:
Isnt this pretty much the same as [here](https://github.com/fsharp/FAKE/blob/master/src/app/Fake.Core.Process/GuardedAwaitObservable.fs) ?

Reply:
As far as i can https://github.com/fsprojects/FSharp.Control.Reactive supports netstandard2.0 now.

Reply:
Yes actually I have no problem with adding that dependency as I'm not able to come up with a solution and give up :)

Reply:
I'm pretty sure we can remove without netstandard1.6 in `Fake.Runtime`